lib/automatic/api_client/client_delegator.rb

```diff
@@ -0,0 +1,21 @@
+module Automatic
+  class ApiClient
+    class ClientDelegator
+      attr_reader :api_client, :delegated_class
+
+      def initialize(api_client, delegated_class)
+        @api_client = api_client
+        @delegated_class = delegated_class
+      end
+
+      def method_missing(method_sym, *arguments, &block)
```

Comment:
If you're going to do a `method_missing` you should also add a `respond_to?` handler. Then you can change line 12 to be `super unless respond_to?(method_sym)`